gateway/1.5/modules/ROOT/pages/policies-included-traffic-management.adoc

```diff
@@ -0,0 +1,154 @@
+= Traffic Management for Multiple Upstream Services Policy
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+:keywords: route weight, traffic management, multiple upstream services
+
+[%autowidth.spread,cols="a,a"]
+|===
+>s| Policy Name | Traffic Management for Multiple Upstream Services
+>s|Summary      | Manages API instance traffic to multiple upstream services from a single consumer endpoint
+>s| First Flex Gateway version available | v1.5.0
+.4+>.^s| Returned Status Codes | No return codes exist for this policy.
+[%autowidth.spread,cols="a,a"]
+|
+|===
+
+[NOTE]
+====
+The following document applies only to Flex Gateway running in Local Mode. To configure multiple upstream services for Flex Gateway running in Connected Mode, see xref:api-manager::create-instance-task-flex.adoc#traffic-management[Multiple Upstream Services for Flex Gateway Running in Connected Mode].
+====
+
+== Summary
+
+Flex Gateway running in Local Mode supports API instances that expose multiple upstream services through a single consumer endpoint.
+
+Flex Gateway manages request traffic by using different routes that can each direct traffic to multiple upstream services. Flex Gateway directs traffic to the routes by using the route order and the individual route's rules. 
+
+API instances with multiple upstream services in Local Mode are configured by using `PolicyBinding` YAML configuration resources. Each `route-weighted` resource defines a single a route. To add additional routes, you can add multiple `route-weighted` resources.
```

Comment:
```suggestion
API instances with multiple upstream services in Local Mode are configured by using `PolicyBinding` YAML configuration resources. Each `route-weighted` resource defines a single route. To add additional routes, add multiple `route-weighted` resources.
```

Reply:
Can you make this text more direct? "Configure API instances with multiple upstream services in Local Mode by..."